controllers/managedocs_controller.go

```diff
@@ -722,6 +731,39 @@ func (r *ManagedOCSReconciler) getDesiredConvergedStorageCluster() (*ocsv1.Stora
 	return sc, nil
 }
 
+func (r *ManagedOCSReconciler) getDesiredConsumerStorageCluster() (*ocsv1.StorageCluster, error) {
+
+	capacityAsString := r.addonParams[consumerRequestedCapacity]
+	onboardingTicket := r.addonParams[consumerOnboardingTicket]
+	storageProviderEndpoint := r.addonParams[consumerStorageProviderEndpoint]
+
+	requestedCapacity, err := resource.ParseQuantity(capacityAsString)
+	if err != nil {
+		return nil, fmt.Errorf("invalid storage cluster capacity value: %v", capacityAsString)
+	}
+
+	currentState := r.storageCluster.Spec.ExternalStorage
+	sc := templates.ConsumerStorageClusterTemplate.DeepCopy()
+	es := &sc.Spec.ExternalStorage
```

Comment:
Can we please stop using shorten names? it is very hard to remember what `es` is referring to when only looking at the code that is using it.

```suggestion
	externalStorageSpec := &sc.Spec.ExternalStorage
```